### PR TITLE
Bitmart markets

### PR DESCRIPTION
**Pull request by samgermain**
On the master branch, it looks like 
```
% bitmart fetchContractMarkets
2021-12-20T11:20:25.260Z
Node.js: v14.17.0
CCXT v1.64.39
bitmart.fetchContractMarkets ()
---------------------------------------------------
[TypeError] this.publicContractGetContracts is not a function

    at fetchContractMarkets       ../../ccxt/ccxt/js/bitmart.js:481       const response = await this.publicContractGetContracts (params);
    at main                       ../../ccxt/ccxt/examples/js/cli.js:246  const result = await exchange[methodName] (... args)            
    at processTicksAndRejections  internal/process/task_queues.js:95                                                                      

this.publicContractGetContracts is not a function
```
 is thrown
 
 It doesn't look like there's many other methods for getting the contract markets either, the api only has the tickers method

### CODE REVIEW

**Comment by kroitor** [comment]
@samgermain why was this closed? Are you going to resubmit it?

**Comment by samgermain** [comment]
> @samgermain why was this closed? Are you going to resubmit it?

It was closed because of the error I listed in my first comment, and I decided I was going to research it more before submitting it, but I couldn't find a method to fetch the contract markets

**Comment by samgermain** [comment]
It looks like they only have the `tickers` endpoint for their contracts api

**Comment by kroitor** [comment]
Related: #10931

File: js/bitmart.js [hunk 1]
```diff
@@ -557,56 +575,73 @@ module.exports = class bitmart extends Exchange {
...
-                    'min': undefined,
-                    'max': undefined,
-                },
-            };
             const contractType = this.safeValue (contract, 'contract_type');
             let future = false;
             let swap = false;
             let type = 'contract';
+            let symbol = base + '/' + quote;
+            const expiry = this.safeValue (contract, 'delive_at');
```
Comment (kroitor) [inline js/bitmart.js:583]: This line is wrong, i think, should use parse8602 here...

**Pull request merged**